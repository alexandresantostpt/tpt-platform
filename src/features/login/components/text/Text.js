import PropTypes from 'prop-types'
import React from 'react'
import styled from 'styled-components'

const StyledText = styled.div`
    color: #858899;
    font-size: 1.75rem;
`

const Text = ({ children }) => <StyledText>{children}</StyledText>

Text.propTypes = {
    children: PropTypes.oneOfType([PropTypes.array, PropTypes.func, PropTypes.object, PropTypes.string]).isRequired
}

export default Text
