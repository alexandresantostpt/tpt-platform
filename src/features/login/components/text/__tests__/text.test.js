import React from 'react'
import renderer from 'react-test-renderer'

import Text from '../Text'

describe('Tests for Text component', () => {
    it('Should rendering without crash', () => {
        const tree = renderer
            .create(
                <Text>
                    <h1>Must be show!</h1>
                </Text>
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})
