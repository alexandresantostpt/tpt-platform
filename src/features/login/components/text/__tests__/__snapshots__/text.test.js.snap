// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Text component Should rendering without crash 1`] = `
<div
  className="Text__StyledText-fa81pq-0 bIShHK"
>
  <h1>
    Must be show!
  </h1>
</div>
`;
